### DISCUSSION TITLE
DIGIT Installation Error

### DISCUSSION THREAD

**Discussion by irad4891** [upyog/UPYOG#696] [Q&A]
Hi Team,
This is Ranjit Singh from Intense Technologies Ltd., Hyderabad.

We are trying an AWS installation of DIGIT Devops build. Repo Link is https://github.com/upyog/UPYOG-DevOps/tree/release 

We were able to provision the Infra. During installation of DIGIT product 2.7 we are encountering the below error.

![DigitInstallError1](https://github.com/upyog/UPYOG/assets/33023683/638d6cdc-2ecd-4b58-8608-27d628c67467)

We have tried changing the apiVersion: extensions/v1beta1 to networking.k8s.io/v1. Still the installation is not sucessfull. Kindly assist in resolution of the same.

**Comment by support-upyog** [comment]
Hi Team,
Please follow below repo for your deployment:
https://github.com/upyog/UPYOG-Training-DevOps

**Reply by support-upyog** [reply]
Hi Team,
Please follow below repo for your deployment:
https://github.com/upyog/UPYOG-Training-DevOps

**Reply by irad4891** [reply]
Hi Team,
We have tried the above mentioned repo. Please find below the error encountered.

![image](https://github.com/upyog/UPYOG/assets/33023683/a93085a0-e87f-4ea5-8d91-ef8564d7da75)

![image](https://github.com/upyog/UPYOG/assets/33023683/019c8383-a023-47be-9862-fe86698521ea)

Kindly assist in possible solution for this. 
Thank You.

**Reply by support-upyog** [reply]
Hi,
Please join Today Technical call @4PM for further discussion.

**Reply by vinothrallapalli-eGov** [reply]
The image pull back error will throw when a particular image is no available in the docker repository. In the **upyogio** docker repository seems those images are not available. Please try with **egovio** docker repository.

**Comment by Gowtham-egov** [comment]
Hello sir,

You can use https://github.com/upyog/UPYOG-DevOps/blob/release-1.27-kubernetes/config-as-code/helm/charts/cluster-configs/templates/ingress/root-ingress.yaml to provision infra. The issue encountered is due to the template that have used for root-ingress.yaml Update the root-ingress.yaml template from above link and deploy.
